app/controllers/content_item_signups_controller.rb

```diff
@@ -35,7 +35,13 @@ def create
 
 private
 
-  PERMITTED_CONTENT_ITEMS = %w[taxon organisation ministerial_role person topical_event].freeze
+  PERMITTED_CONTENT_ITEMS = %w[taxon
+                               organisation
+                               ministerial_role
+                               person
+                               topical_event
+                               service_manual_topic
+                               service_manual_service_standard].freeze
```

Comment:
This needs to be rebased following Kelvin's [change to support topics](https://github.com/alphagov/email-alert-frontend/pull/789)

Reply:
Sure - I've added another commit after doing that. Can you take another look?